app/controllers/users.js

```diff
@@ -53,5 +54,18 @@ module.exports = {
         logger.error(`DB: ${error.errors[0]}`);
         res.status(500).json({ error: error.errors[0].message });
       });
+  },
+  usersList(req, res, next) {
+    let page = parseInt(req.query.page) || 1;
+    page = page > 0 ? page : 1;
+    const pageSize = 10;
+    User.findAll({ offset: pageSize * (page - 1), limit: pageSize })
+      .then(users => {
+        res.status(200).json({ users, page });
```

Comment:
We should also send some information about the total pages or the total amount of elements, so the front or mobile can display de UI correctly.

Reply:
Done